submissions/UA-girl/task_js_oop_exersise/index.js

```diff
@@ -0,0 +1,47 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/UA-girl/a-tiny-JS-world
+   Web app: https://ua-girl.github.io/a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+
+class Creature {
+    constructor(species, name, age, gender, legs, say, friends = []) {
+        this.species = species;
+        this.name = name;
+        this.age = age;
+        this.gender = gender;
+        this.say = say;
+        this.legs = legs
+        this.friends = friends.length > 0 ? friends : [];
```

Comment:
This situation is handled by default value in constructor arguments.
Perhaps we'd rather need to handle a situation when `friends` argument is not an array at all.